main/api/src/mill/api/ClassLoader.scala

```diff
@@ -90,7 +90,9 @@ object ClassLoader {
         )
         retry {
           try os.copy(os.Path(Export.rt()), java90rtJar, createFolders = true)
-          catch { case e: FileAlreadyExistsException => /* someone else already did this */}
+          catch {
+            case e: FileAlreadyExistsException => /* someone else already did this */
+          }
         }
```

Comment:
@lihaoyi this file also got same changes in my PR while fixing linting error should i remove the changes from there or rebasing and merge will automatically fix this